src/Card/Card-styled.js

```diff
@@ -114,16 +114,25 @@ const StyledCardImage = styled.img`
   display: block;
 
   ${props =>
-    props.wide &&
-    css`
-      width: auto;
-      min-width: 100%;
-      max-width: none;
-      height: 100%;
-      margin: 0;
-      display: block;
-      position: absolute;
-    `};
+    props.wide
```

Comment:
Styles that are not conditional based on a prop should be defined above the props arrow function, eg line 115